app/operations/transfers/to_enroll_batch.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require 'dry/monads'
+require 'dry/monads/do'
+require 'aca_entities/serializers/xml/medicaid/atp'
+require 'aca_entities/atp/transformers/cv/family'
+module Transfers
+  # Transfer an account from ACES to enroll
+  class ToEnrollBatch
+    send(:include, Dry::Monads[:result, :do, :try])
+    # @param [String] Application ID from ACES
+    # @return [Dry::Result]
+    def call(transfer_id, itransfers = [])
+      transfers = yield get_transfers(transfer_id, itransfers)
+      payload = yield get_payload(transfers)
+      transformed_params = yield transform_params(payload, transfers)
+      initiate_transfer(transformed_params, transfers)
+      Success([transfers, transformed_params])
+    end
+
+    private
+
+    def get_transfers(transfer_id, itransfers)
+      return Success(itransfers) if itransfers.any?
+      inbound_transfers = Aces::InboundTransfer.all.select(&:waiting_to_transfer?).select {|t| t.external_id == transfer_id }
+      return Failure("no transfers found") unless inbound_transfers
```

Comment:
Looks like inbound_transfers will always be an array here, which will evaluate as true even if empty, so I don't think this will return Failure when no transfers are found.  Could use 'unless inbound_transfers.present?' or 'if inbound_transfers.empty?' to get the expected behavior.